tests/PHPStan/Reflection/data/returns-by-reference.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace ReturnsByReference;
```

Comment:
I am not aware of a php-src native method/function which returns by reference, therefore I tested the userland class/method/function cases only

Reply:
There probably isn't one, right.